Client/Frontend/Browser/TabDisplayManager.swift

```diff
@@ -890,8 +890,6 @@ extension LegacyTabDisplayManager: TabManagerDelegate {
     }
 
     func tabManager(_ tabManager: TabManager, didAddTab tab: Tab, placeNextToParentTab: Bool, isRestoring: Bool) {
```

Comment:
nit: seems like we can also remove the param now
```suggestion
    func tabManager(_ tabManager: TabManager, didAddTab tab: Tab, placeNextToParentTab: Bool) {
```

Reply:
I initially thought so, but l've found this is [used in TabManager](https://github.com/mozilla-mobile/firefox-ios/blob/3b6eca3873996a7e8d5981d85d8c9c5809888f00/Client/TabManagement/Legacy/LegacyTabManager.swift#L461) to pass up the chain to BVC if the tabManager has done restoring the tabs, and then update the tab count. So I don't think we can clean this one up sadly